### ISSUE TITLE
Containerized QLayout drawer bug

### ISSUE THREAD

**Issue by nicholaszuccarelli** [quasarframework/quasar#4075]
I am having an issue with my containerized QLayout:

https://i.gyazo.com/c46deee1b42cda07362306c78c30f82a.mp4

So my drawer starts off in mobile mode, but if I open a dialog and then close the dialog, the drawer reverts to desktop mode - which it shouldn't.

**Comment by pdanpdan** [comment]
Please provide a fiddle with code.

**Comment by nicholaszuccarelli** [comment]
Unfortunately I wasn't able to replicate it in Codepen because it uses the vue-router (for my bug to happen) and my skills on Codepen aren't that great :P

I was actually able to solve the problem by adding persistent to the drawer (to keep it open on route change).

If it aids in any way, if the drawer isn't persistent and you click on a link inside of the drawer, the drawer WILL close (which is fine), but if I navigate back to the previous route (where I clicked the link on), the drawer reopens in "desktop" view mode, as per the video I supplied.

I'll close the issue for now seeing as I personally solved it for my situation.
